packages/suite-data/files/translations/en.json

```diff
@@ -452,7 +455,7 @@
   "TR_WALLET": "Accounts",
   "TR_WALLET_SELECTION_HIDDEN_WALLET": "Hidden wallet",
   "TR_WIPE_DEVICE_TEXT": "Resetting the device removes all its data. Reset your device only if you have a safe offline backup of your recovery seed, which allows you to restore your funds.",
-  "TR_WIPING_YOUR_DEVICE": "Factory Reset wipes the device memory, erasing all information including the Recovery Seed and PIN. Only perform a Factory Reset if you have a safe offline backup of your Recovery Seed, which allows you to restore your funds.",
+  "TR_WIPING_YOUR_DEVICE": "Factory reset wipes the device memory, erasing all information including the recovery seed and PIN. Only perform a factory reset if you have a safe offline backup of your recovery seed, which allows you to restore your funds.",
```

Comment:
I think we should somehow unify - "backup" and "recovery seed" wording, do people know, that we are talking about the same thing?